src/frontend/data/stores/useMessagesStore/index.ts

```diff
@@ -0,0 +1,25 @@
+import { DateTime } from 'luxon';
+import create from 'zustand';
+
+export type ChatMessageType = {
+  dateTime: DateTime;
```

Comment:
`dateTime` does not explain what it means. What is this `dateTime` ?

Reply:
dateTime refers to the sending datetime of the message, so i renamed it sendingDateTime
see [bb476d5c7dd392ea74c9a6e0ea275d562cc71085](https://github.com/openfun/marsha/pull/1247/commits/bb476d5c7dd392ea74c9a6e0ea275d562cc71085)

Reply:
`sended_at` can be enough ?

Reply:
sent_at then 😉